scripts/generate_pinval/pyproject.toml

```diff
@@ -1,8 +1,6 @@
 [project]
 name = "generate-pinval"
 version = "0.1.0"
-description = "Add your description here"
-readme = "README.md"
```

Comment:
These two attributes aren't required, and they're not really adding anything right now.